api/v1/kube/kube_api.proto

```diff
@@ -12,8 +12,8 @@ service KubeAPI {
 
 message GetClusterInfoRequest {}
 message GetClusterInfoResponse {
-  string pods_cidr = 1;
-  string service_cidr = 2;
+  repeated string pods_cidr = 1;
+  repeated string service_cidr = 2;
```

Comment:
This is a breaking change but if kvisor agent is the only client of the kvisor controller API we should be fine